src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/form/main.js

```diff
@@ -28,6 +28,15 @@ define(['app-config', 'config', 'sulucontent/components/content/preview/main'],
             };
         },
 
+        collaboration: function() {
+            return {
+                autoStart: false,
```

Comment:
Why do we set `autoStart` here to false? Actually we could get rid of this setting, or is there any use case for it? If not necessary I would always assign an object to `collaboration` and just start it. From what I see [here](https://github.com/sulu/sulu/pull/2201/files#diff-c124d8cf9d9f11172e744e9e64c33739R196) it is started anyway if the collaboration component is registered, so I don't see a point for this flag.